.github/workflows/_update-deps.yaml

```diff
@@ -44,14 +44,15 @@ jobs:
       - name: Set Git config
         run: |
           git config --global --add safe.directory ${GITHUB_WORKSPACE}
+      - uses: vdaas/vald-client-ci/.github/actions/dependabot@fix/add-directory-name
```

Comment:
```suggestion
      - uses: vdaas/vald-client-ci/.github/actions/dependabot@main
```

Reply:
Others LGTM

Reply:
Thank you for catching my mistake 🙏 
I apply your suggestion 👍